nautobot/core/views/mixins.py

```diff
@@ -750,6 +777,18 @@ class ObjectEditViewMixin(NautobotViewSetMixin, mixins.CreateModelMixin, mixins.
     UI mixin to create or update a model instance.
     """
 
+    def extra_message_context(self, obj):
+        """
+        Context variables for this extra message.
+        """
+        return {}
+
+    def extra_message(self, **kwargs):
+        """
+        Append extra message at the end of create or update success message.
+        """
+        return ""
```

Comment:
Why not just make another separate call to `messages.success()`?

Reply:
I am running into formatting issues when I try to call `messages.success()` directly in the function.

Reply:
There could be cases where you would override `extra_message` and not `extra_message_context`. I do not think so since all old models are okay with the default message. And we can reasonably assume that new models implemented in the future will be using `NautobotUIViewSet` and `ObjectEditViewMixin`.